<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mystic</class>
 <widget class="QMainWindow" name="Mystic">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>682</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Mystic</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>600</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>381</width>
      <height>531</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::CopyAction</enum>
    </property>
    <property name="resizeMode">
     <enum>QListView::Fixed</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>600</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>600</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string>Conversion Mode</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="RAWradioButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>RAW Convert</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="MLVradioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MLV Convert</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="dualIsoradioButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Dual Iso Convert</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>161</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QCheckBox" name="indexBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>268</y>
        <width>161</width>
        <height>44</height>
       </rect>
      </property>
      <property name="text">
       <string>Index Files (Fixes Crashing)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="DNGBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>18</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>To DNG</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="RAWBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>70</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>To RAW</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="stripeBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Vertical Stripes Fix
       (5D III)</string>
      </property>
      <property name="shortcut">
       <string extracomment="Fixes vertical lines formed on some canon models. DNG only."/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="metaBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>328</y>
        <width>91</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Metadata LOG</string>
      </property>
     </widget>
     <widget class="QComboBox" name="NConventcomboBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentText">
       <string>NAME . ######</string>
      </property>
      <item>
       <property name="text">
        <string extracomment=".">NAME . ######</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string extracomment="_">NAME _ ######</string>
       </property>
      </item>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>79</y>
        <width>121</width>
        <height>121</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QCheckBox" name="MLV_DualISOcheckBox">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>10</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Dual ISO (ALPHA)</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="ChromacheckBox">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>38</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Chroma Smooth</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>90</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>No Smooth</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="ChromacomboBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string notr="true">2x2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">3x3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">5x5</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QCheckBox" name="FolderscheckBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Sub Folders</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QCheckBox" name="stripeBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Vertical Stripes Fix
       (5D III)</string>
      </property>
      <property name="shortcut">
       <string extracomment="Fixes vertical lines formed on some canon models. DNG only."/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="destinationButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>540</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Destination</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="destDirectory">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>570</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
